.github/workflows/cla.yml

```diff
@@ -0,0 +1,32 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
```

Comment:
I don't think we need this comment, it's from the contributor-assistant/github-action README, but we can explain why we need the permissions (and link the permissions docs, a nice touch I think)

```suggestion
# https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs
# https://github.com/contributor-assistant/github-action needs permissions to
# - re-run workflows
# - comment in pull requests
# - update commit statuses 
```